sdk/spring/spring-cloud-azure-starter-monitor/src/main/java/com/azure/monitor/applicationinsights/spring/OpenTelemetryVersionCheckRunner.java

```diff
@@ -19,7 +19,7 @@ public class OpenTelemetryVersionCheckRunner implements CommandLineRunner {
     /**
      * OpenTelemetry version of the starter
      */
-    public static final String STARTER_OTEL_VERSION = "1.38.0";
```

Comment:
this qualifies as a behavior-breaking change, which is ok for beta package.  But I wonder if this constant needs to be public - do we expect end users to use it and if not, then can we make it private or move it to implementation package?

It's ok to do it in a separate PR later, but before the package is GA-ed.